src/app/shared/interfaces/target-with-files.ts

```diff
@@ -0,0 +1,3 @@
+export interface TargetWithFiles extends EventTarget {
```

Comment:
Look at my comment about using HtmlInputElement, that way this interface is redundant and can be removed